BoxItScript.js

```diff
@@ -0,0 +1,41 @@
+const args = process.argv.slice(2);
+const words = args;
+function drawLine(num) {
+    let res = '━'.repeat(num);
+    return res;
+}
+function drawTopBorder(num) {
+    let res = '┎' + drawLine(num) + '┒';
+    return res;
+}
+function drawMiddleBorder(num) {
+    let res = '┣' + drawLine(num) + '┫';
+    return res;
+}
+function drawBottomBorder(num) {
+    let res = '┗' + drawLine(num) + '┛';
+    return res;
+}
+function drawBarsAround(str) {
+    let res = '┃' + str + '┃';
+    return res;
+}
+function boxIt(arrstr) {
+    let res = [];
+    let maxstr = arrstr[0].length;
```

Comment:
When possible try to use camelcase
eg. `arrStr` instead of `arrstr`

Also descriptive names are easier to read after a long absence or by other team members who are not familiar with your code.

eg. `strings` instead of `arrstr` .--- `let string of strings` ---- `maxLength` instead of `maxstr`